02-peer-protocol.md

```diff
@@ -543,9 +565,18 @@ progress is made, even if only by a single satoshi at a time. To avoid
 keeping state and to handle the corner case, where fees have shifted
 between disconnection and reconnection, negotiation restarts on reconnection.
 
-Note there is limited risk if the closing transaction is
-delayed, but it will be broadcast very soon; so there is usually no
-reason to pay a premium for rapid processing.
+In the `option_simplified_commitment` case, the fees on the commitment
+transaction itself are minimal (it is assumed that a child transaction will
+supply additional fee incentive), so that forms a floor for negotiation.
+[BOLT #3](03-transactions.md#fee-calculation), gives 282 satoshis (1116
+weight, 254 `feerate_per_kw`).
```

Comment:
Shouldn't this be 253 `feerate_per_kw` instead of 254? 254 doesn't seem consistent with the calculation (yields 283 satoshis) and with the previous logic (253 for all commitment transactions).